app/support/reservations/validations.rb

```diff
@@ -23,7 +23,7 @@ module Reservations::Validations
     validate :duration_is_interval
 
     validates :category,
-              inclusion: { in: -> (r) { r.class::CATEGORIES }, allow_blank: false },
+              inclusion: { in: -> (r) { r.class::CATEGORIES }, allow_blank: true },
```

Comment:
For the base `Reservation` model, I'd leave this as-is. I'd put the category presence validation in `AdminReservation`, much like it is for `OfflineReservation`.